docs/content/ingestion/schema-design.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: doc_page
+---
+
+# Schema Design
+
+This page is meant to assist users in designing a schema for data to be ingested in Druid. Druid intakes denormalized data 
+and columns are one of three types: a timestamp column, a dimension, or a measure (or a metric/aggregator as they are 
+known in Druid). This follows the [standard naming convention](https://en.wikipedia.org/wiki/Online_analytical_processing#Overview_of_OLAP_systems) 
+of OLAP data.
+
+Typical production tables (or datasources as they are known in Druid) have less than 100 dimensions and less 
+than 100 metrics, although, based on user testimony, datasources with thousands of dimensions have been created. At the 
+time of writing, we have not benchmarked the upper limits of how many columns Druid is capable of.
+
+Dimension values in Druid can either be a single valued String, or an array of String values. Metric columns can be 
+integers, longs, floats, or in a user defined 'complex' format. Metric values may be [rolled-up](../design/index.html) 
```

Comment:
maybe a clearer thing would be to talk about how they're used rather than how they're stored? so like,

- Every row in Druid has a timestamp. Data is always partitioned by time, and every query has a time filter. Query results can also be broken down by time buckets like minutes, hours, days, and so on.
- Dimensions are fields that can be filtered on or grouped by. They are always either single Strings or arrays of Strings.
- Metrics are fields that can be aggregated. They are often stored as numbers (integers or floats) but can also be stored as complex objects like HyperLogLog sketches or approximate histogram sketches.